web/styles/settings.css

```diff
@@ -228,7 +228,7 @@ h3,
 }
 
 #uploaded_files_table > tr > td:nth-of-type(1),
-.upload-file-name {
+#attachments-settings .upload-file-name {
```

Comment:
Do we also need to change the selectors like upload-size, upload-date, upload-mentioned-in?

Reply:
It's probably fine, those don't seem super likely to have collisions.

Reply:
`> tr > td:nth-of-type(1)` is evil though, and the sort of thing that may slow painting in general. But we've got a #frontend thread about a broader sweep project to clean that sort of thing up.